sixMans/sixMans.py

```diff
@@ -1333,6 +1333,10 @@ async def _pre_load_games(self, guild: discord.Guild):
                 game.orange = set([guild.get_member(x) for x in value["Orange"]])
                 game.roomName = value["RoomName"]
                 game.roomPass = value["RoomPass"]
+                try:
+                    game.info_message = await game.textChannel.fetch_message(value["InfoMessage"])
+                except:
+                    pass
```

Comment:
I think if we fail to get an info message, we could consider cancelling the game due to an issue, or checking the state of the game otherwise -- teams, players, etc. 

But this is good for now. Perhaps we could add a TODO